### DISCUSSION TITLE
Error d:d:c

### DISCUSSION THREAD

**Discussion by IsmaelARassool** [doctrine/DoctrineBundle#1678] [Q&A]
Hello I'm on windows, with xampp, i'm trying to create a database but I have this problem
![image](https://github.com/symfony/symfony/assets/133125183/ce342406-1d89-4bb0-9bf7-64ac23fb9423)
I already uncommented the extensions pdo and mysql
here is my .env.local
![image](https://github.com/symfony/symfony/assets/133125183/ec614988-a3ac-4f96-b0b2-0d4363c73853)
I also tryed to uninstall everything and reinstall it, tryed to create the database myself and make a d:s:u but same error
Thanks for reading. Have a nice day !

**Comment by ghost** [comment]
please provide exception stack trace (run command with `-vvv` flag). Also please provide doctrine.yaml content and output from `php -m`. phpinfo() output would help as well.

**Reply by IsmaelARassool** [reply]
this is with -vvv
![image](https://github.com/doctrine/DoctrineBundle/assets/133125183/da94e72c-522e-4ff4-bf6b-58566f9c36c4)
![image](https://github.com/doctrine/DoctrineBundle/assets/133125183/2a899eea-7ecd-4c2e-9e63-f7290436edb3)
![image](https://github.com/doctrine/DoctrineBundle/assets/133125183/6a15c90d-3654-4998-8ea3-e3e9405b9bb9)
this is my doctrine.yaml
![image](https://github.com/doctrine/DoctrineBundle/assets/133125183/7ef651f4-683a-4651-9cdb-fc777a394d79)
and the php -m
![image](https://github.com/doctrine/DoctrineBundle/assets/133125183/4c5c08c3-df75-4119-8cac-448accf97a0a)

**Reply by stof** [reply]
To be able to use the PDO mysql driver, you need to load the `pdo_mysql` extension, which does not appear in your screenshot

**Reply by ghost** [reply]
I'm not familiar with `mysqlnd` extension. Try enabling `pdo_mysql`

**Reply by IsmaelARassool** [reply]
here is phpinfo()
![image](https://github.com/doctrine/DoctrineBundle/assets/133125183/d3f1ab4d-24af-4f11-81a5-acdb53ed8a4d)
pdo_mysql is already enabled
![image](https://github.com/doctrine/DoctrineBundle/assets/133125183/5a415615-e659-4c10-a097-c52c334412cb)

**Reply by ghost** [reply]
Are you sure this is the right php.ini? I can see other extensions that are uncommented but do not appear in `php -m`

**Reply by stof** [reply]
The screenshot is far from being the full phpinfo, and it does not show the part listing the loaded PDO drivers.

But your `php -m` output was not showing `pdo_mysql` in the list so I doubt it is actually loaded. Either you have different php.ini for the CLI and the webserver (and you need to check both of them to see whether it is there) or PHP fails to load the extension (which should display a warning in the CLI when starting the process, so I think it is rather a case of not having it at all in the CLI config)

**Reply by ghost** [reply]
Please put the following code at the top of the `bin/console` file:
```php
echo phpinfo();
exit(0);
```

and copy past output here.

**Reply by IsmaelARassool** [reply]
this is the php.ini from my C://xampp/php/
but i also have a php.ini in C://web/php/

**Reply by IsmaelARassool** [reply]
I think it's good I do the same uncomment from C://xampp/php/php.ini to C://web/php/php.ini and now my database is created
![image](https://github.com/doctrine/DoctrineBundle/assets/133125183/3eea71ba-6ba1-4a74-948f-a34140f22346)
thank you !
